android/res/layout/view_ad_menuitem.xml

```diff
@@ -81,6 +74,7 @@
         android:layout_centerVertical="true"
         android:layout_marginRight="12dp"
         android:src="@drawable/ad_menu_speaker"
+        android:tint="@color/basic_white"
```

Comment:
@aldenml why is that, just curious? Tested on devices with lower API and shows fine. I guess we can also try to use <android.support.v7.widget.AppCompatImageView> instead of <ImageView> but from what I read this should not be necessary. 
I can change it back and use a regular icon, but that means another asset. We also use it to tint the nav drawer icons now. Do we need to remove it?